packages/web-components/fast-element/package.json

```diff
@@ -65,6 +65,14 @@
     "./metadata": {
       "types": "./dist/dts/metadata.d.ts",
       "default": "./dist/esm/metadata.js"
+    },
+    "./testing": {
+      "types": "./dist/dts/testing/exports.d.ts",
+      "default": "./dist/esm/testing/exports.js"
+    },
+    "./di": {
```

Comment:
What is `./di` ?

Reply:
This stands for "dependency injection". I debated spelling that all out. In the end, I decided to go with the abbreviation because the longer string "dependency-injection" could negatively affect bundle size if imports are frequently needed, which I think could be likely in an app that uses this feature.